### PR TITLE
C350710 Check date and time -- loans (firebird) (TaaS)

### PR DESCRIPTION
**Pull request by ernestyedigaryan1**
[C350710](https://foliotest.testrail.io/index.php?/cases/view/350710)

[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4603/allure/)

### CODE REVIEW

**Review by dnovikau — COMMENTED** [review]
I propose to move the test to a separate file as it has it's own preconditions.

Test should be added to 'cypress/e2e/circulation-log' folder

**Review by Scorpibear — APPROVED** [review]

File: cypress/support/fragments/circulation-log/searchPane.js [hunk 1]
```diff
@@ -210,6 +212,10 @@ export default {
     cy.do([dropdownButton.click(), DropdownMenu().find(Button()).click()]);
   },
 
+  goToItemDetails(barcode) {
+    cy.do(Link(including(barcode)).click());
+  },
+
```
Comment (re-jas) [inline cypress/support/fragments/circulation-log/searchPane.js:218]: We already have a method for this, please use:
`SearchPane.findResultRowIndexByContent(content).then((rowIndex) => { SearchResults.clickOnCell(content, row );});`

File: cypress/e2e/circulation-log/loan-date-and-time.cy.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      InventoryInstances.createFolioInstancesViaApi({
+        folioInstances: testData.folioInstances,
+        location,
+      });
+    });
+    cy.createTempUser([
+      Permissions.circulationLogAll.gui,
+      Permissions.uiInventoryViewInstances.gui,
+      Permissions.uiUsersViewLoans.gui,
+      Permissions.inventoryAll.gui,
```
Comment (nayimovag) [inline cypress/e2e/circulation-log/loan-date-and-time.cy.js:40]: do you need inventory all permissions?

Comment (nayimovag) [reply cypress/e2e/circulation-log/loan-date-and-time.cy.js:40]: add it to the permissions.js 
internal: 'ui-inventory.item.delete',  gui: 'Inventory: View, create, edit, delete items'

Comment (ernestyedigaryan1) [reply cypress/e2e/circulation-log/loan-date-and-time.cy.js]: done

**Review by nayimovag — APPROVED** [review]

**Pull request merged**